src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -13,30 +13,81 @@
 
 package com.hpe.adm.octane.ideplugins.integrationtests;
 
+import com.google.api.client.json.Json;
 import com.google.inject.Guice;
 import com.google.inject.Inject;
 import com.google.inject.Injector;
+import com.hpe.adm.nga.sdk.Octane;
+import com.hpe.adm.nga.sdk.authentication.Authentication;
+import com.hpe.adm.nga.sdk.authentication.SimpleUserAuthentication;
+import com.hpe.adm.nga.sdk.model.EntityModel;
+import com.hpe.adm.nga.sdk.network.OctaneHttpClient;
+import com.hpe.adm.nga.sdk.network.OctaneHttpRequest;
+import com.hpe.adm.nga.sdk.network.OctaneHttpResponse;
+import com.hpe.adm.nga.sdk.network.OctaneRequest;
+import com.hpe.adm.nga.sdk.network.google.GoogleHttpClient;
 import com.hpe.adm.octane.ideplugins.integrationtests.util.EntityGenerator;
 import com.hpe.adm.octane.ideplugins.integrationtests.util.PropertyUtil;
+import com.hpe.adm.octane.ideplugins.integrationtests.util.WorkSpace;
+import com.hpe.adm.octane.ideplugins.services.EntityService;
+import com.hpe.adm.octane.ideplugins.services.connection.ConnectionSettings;
 import com.hpe.adm.octane.ideplugins.services.connection.ConnectionSettingsProvider;
+import com.hpe.adm.octane.ideplugins.services.connection.HttpClientProvider;
 import com.hpe.adm.octane.ideplugins.services.connection.OctaneProvider;
 import com.hpe.adm.octane.ideplugins.services.di.ServiceModule;
+import com.hpe.adm.octane.ideplugins.services.filtering.Entity;
+import com.hpe.adm.octane.ideplugins.services.util.ClientType;
+import com.sun.org.apache.xpath.internal.SourceTree;
+import org.apache.commons.logging.impl.Log4JLogger;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import sun.net.www.http.HttpClient;
+
+import java.lang.annotation.Annotation;
+import java.lang.annotation.Target;
+import java.lang.reflect.AnnotatedType;
+import java.util.Collection;
 
 /**
  * Enables the use of the {@link Inject} annotation
  */
 public abstract class IntegrationTestBase {
 
+
+    final Log4JLogger logger = new Log4JLogger();
+
     protected EntityGenerator entityGenerator;
 
+    ConnectionSettingsProvider connectionSettings;
+
     @Before
-    public void setup () {
+    public void setup() {
 
-        ConnectionSettingsProvider connectionSettings = PropertyUtil.readFormVmArgs() != null ?
+        connectionSettings = PropertyUtil.readFormVmArgs() != null ?
```

Comment:
Since creating a workspace for the test, the user should not have to provide the default one, the file does not need a workspace field, instead the default workspace should be detected by the tests. In case there is no workspace in the specified shared spache to use as default. The base should just create one